app/Main.hs

```diff
@@ -26,12 +28,35 @@ initScreen width height = do
         SDL.setWindowTitle window s
     return $ SDLContext window renderer
 
+getWindow :: SDLContext -> SDL.Window
+getWindow (SDLContext window _) = window
+
+getRenderer :: SDLContext -> SDL.Renderer
+getRenderer (SDLContext _ renderer) = renderer
+
+arial :: String
+arial = "../assets/arial.ttf"
+
 main :: IO ()
 main = do
     ctx <- initScreen 800 600
     ptrEvent <- malloc
-    draw ctx
-    eventLoop ptrEvent ctx
+    --draw ctx
+    
+    TTF.withInit $ do
+        font <- TTF.openFont arial 150
+        textSurface <- TTF.renderUTF8Solid font "test" (SDL.Color 255 255 255 0)
+        textTexture <- SDL.createTextureFromSurface (getRenderer ctx) textSurface
+        loop ptrEvent ctx textTexture 
```

Comment:
you'll want to add a new case [here](https://github.com/forestbelton/orb/blob/master/src/Paint.hs#L11),
and then handle it in `paintCmd`.